frontend/src/components/drawers/ConfigDrawer/tabs/tab-content-mcp.tsx

```
@@ -0,0 +1,322 @@
+    try {
+      // Parse the current mcpCode
+      let currentConfig = {};
+      try {
+        currentConfig = JSON.parse(mcpCode);
+      } catch (e) {
+        // If current code isn't valid JSON, start with an empty object
+        currentConfig = {};
+      }
+
+      // Check if this config is already selected
+      if (typeof selectedConfig.slug === 'string' && currentConfig.hasOwnProperty(selectedConfig.slug)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)